src/decanter/core/jobs/experiment.py

```diff
@@ -94,6 +94,9 @@ def create(cls, exp_id, name=None):
     def update_result(self, task_result):
         """Update Job's attribute from Task's result."""
         self.get_best_model()
+        self.recommend_model_id = {rec_dict['evaluator']: rec_dict['model_id'] \
```

Comment:
I guess that it's my mistake.. this data has been store at 'recommendations ' and `models`. Those data might be updated in task.CoreTask.update_task_response

Reply:
fix, i use experiment.recommendations to replace the self.recommend_model_id